doc/sphinx/source/howto/tune-for-deployment.rst

```diff
@@ -0,0 +1,181 @@
+.. tune-for-deployment.rst
+
+
+############################################
+Fine-tune code deployment to nGraph devices  
+############################################
```

Comment:
Fine-tune code for deployment on nGraph devices

Reply:
Hmmm.  That's not really what this is though.  This is various system-level config settings people can tune to get better perf out of CPU... audience is framework (bridge code) developers, not data scientists running the models.  We do have access to Xeon-level perf via NUMA config (mentioned in the article) that can be built into a framework/kernel.

Reply:
How about "Fine-tune topologies for deployment to nGraph devices" or something like that. The problem is that "fine-tune" is binding to "code" rather than "code deployment" so it feels like something is missing.

Reply:
I think "Optimize for device characteristics" would be a better heading.